src/audio/mux/mux.c

```diff
@@ -530,15 +530,18 @@ static int mux_process(struct processing_module *mod,
 	int frames;
 	int sink_bytes;
 	int source_bytes;
-	int i;
```

Comment:
Should add to commit message also Fixes: c399624 ("Audio: Mux: Convert mux and demux to module adapter")